src/storage/StorageServiceHandler.cpp

```diff
@@ -249,10 +250,15 @@ StorageServiceHandler::future_rebuildEdgeIndex(const cpp2::RebuildIndexRequest&
 
 folly::Future<cpp2::LookUpIndexResp>
 StorageServiceHandler::future_lookUpIndex(const cpp2::LookUpIndexRequest& req) {
+    folly::Executor* executor = nullptr;
```

Comment:
There is a ioThreadPool called `readerPool_`, you can reuse it.

Reply:
I've fixed it, and added a switch to decide whether to traversal concurrently , please check it.